.bazelrc

```diff
@@ -0,0 +1,15 @@
+build --enable_platform_specific_config
+
+# linux
+build:linux --cxxopt=-std=c++17
+# See https://bytecodealliance.github.io/wasmtime/c-api/
+build:linux --action_env=BAZEL_LINKOPTS=-lm:-lpthread:-ldl
```

Comment:
Would `--linkopt` work here as well, or do we really need `BAZEL_LINKOPTS`?

Reply:
yeah linkopt actually works!